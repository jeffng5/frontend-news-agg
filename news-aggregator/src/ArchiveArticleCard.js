import React from 'react'
import './css/App.css'

const ArchiveArticleCard = ({title, url, description, author}) => {


// returning the passed data thru structured html     
return (
    <>
    <h6 className= 'divider-card'>
<h6 className='archive-card'>Title: {title} </h6>
<h6 className='archive-card'>Description: {description}</h6>
<h6 className = 'archive-card'><a href={url}>{url}</a></h6> 
<h6 className='archive-card-1'>By : {author}</h6>
</h6>
</>
)
}

export default ArchiveArticleCard