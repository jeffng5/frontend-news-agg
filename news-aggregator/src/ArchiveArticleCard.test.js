import React from 'react';
import { render } from '@testing-library/react';
import ArchiveArticleCard from './ArchiveArticleCard'

it('renders without crashing', function() {
  render(<ArchiveArticleCard />);
})

//snapshot test
it("matches snapshot", function() {
    const {asFragment} = render(<ArchiveArticleCard />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('testing component', ()=> {
    const {getByText} = render( <ArchiveArticleCard />);
    getByText('Title:')
})