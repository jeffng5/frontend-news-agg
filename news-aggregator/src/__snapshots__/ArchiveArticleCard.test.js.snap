// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h6
    class="divider-card"
  />
  <h6
    class="archive-card"
  >
    Title:  
  </h6>
  <h6
    class="archive-card"
  >
    Description: 
  </h6>
  <h6
    class="archive-card"
  >
    <a />
  </h6>
  <h6
    class="archive-card-1"
  >
    By : 
  </h6>
</DocumentFragment>
`;
